exporter/awsemfexporter/config.go

```diff
@@ -132,9 +138,39 @@ func (config *Config) Validate() error {
 		return errors.New("invalid value for retention policy.  Please make sure to use the following values: 0 (Never Expire), 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 2192, 2557, 2922, 3288, or 3653")
 	}
 
+	tagInputReturnVal := isValidTagsInput(config.Tags)
+	if tagInputReturnVal != "Valid" {
+		return errors.New(tagInputReturnVal)
+	}
+
 	return nil
 }
 
+// Check if the tags input is valid
+func isValidTagsInput(input map[string]*string) string {
```

Comment:
I wonder if instead of duplicating code here we should move these functions to https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/internal/aws/cwlogs in a `utils.go` or `config.go` module.

Reply:
I agree maybe we should move these duplicate functions somewhere but I wasn't sure where the best place to put it would be.

Reply:
I would put it here https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/internal/aws/cwlogs

It is a safe place to put this because it is a internal module.

Reply:
Put into internal module in my latest commit.